src/main/java/frc/robot/commands/MoveToPose.java

```diff
@@ -0,0 +1,20 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Robot;
+import frc.robot.subsystems.Drivetrain;
+
+public class MoveToPose extends CommandBase{
```

Comment:
This is being done with the swerve team, please don't add unimplemented commands.

Reply:
I just wanted to put something there, for how to use selectedNode.

Reply:
I would add that to the comments or design doc. It just seems inefficient to create three classes to do nothing but document a function. selected node should probably be a function btw.

Reply:
It can be deleted when someone has a better version.